camera/camera.go

```diff
@@ -0,0 +1,36 @@
+components {
```

Comment:
This file is not used. It should be removed.

Reply:
Done. Camera object removed